phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -866,4 +871,18 @@ public static void setTenantId(Configuration configuration, String tenantId){
         configuration.set(MAPREDUCE_TENANT_ID, tenantId);
     }
 
+    public static void setMRSnapshotManagedExternally(Configuration configuration, Boolean isSnapshotRestoreManagedExternally) {
+        Preconditions.checkNotNull(configuration);
+        Preconditions.checkNotNull(isSnapshotRestoreManagedExternally);
+        configuration.set(MANAGE_MR_SNAPSHOT_RESTORE_EXTERNALLY,
+                String.valueOf(isSnapshotRestoreManagedExternally));
+    }
+
+    public static boolean isMRSnapshotManagedExternally(final Configuration configuration) {
+        Preconditions.checkNotNull(configuration);
+        boolean isSnapshotRestoreManagedInternally =
```

Comment:
Typo in the variable name: `isSnapshotRestoreManagedInternally`. Should it be isSnapshotRestoreManaged*Ex*ternally ?